### DISCUSSION TITLE
Cant see the transactions in my wallet!

### DISCUSSION THREAD

**Discussion by Newpolax** [btcpayserver/btcpayserver#5724] [Technical Support] [answered]
Since two days ago, I can not see the recent transactions in my wallet, tested both on desktop and my phone. Transactions and balance is different in BTCpay server and my wallet. Any idea?
I even tried importing the key to another wallet, still the same!

**Comment by dennisreimann** [comment, marked as answer]
It might be [the gap limit problem](https://docs.btcpayserver.org/FAQ/Wallet/#the-gap-limit-problem). Can you check?
